fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java

```diff
@@ -1143,7 +1143,8 @@ public Map<SlotRef, Expr> getPointQueryEqualPredicates() {
 
     public boolean isPointQuery() {
```

Comment:
It's complex and invasive to OlapScanNode.

Reply:
isPointQueryShortCircuit and isShortCircuitQuery are confusing.

Reply:
will be removed